components/climate/tuya.rst

```diff
@@ -11,23 +11,33 @@ Tuya climate requires a :doc:`/components/tuya` to be configured.
 
 .. code-block:: text
 
-    [22:03:11][C][tuya:023]: Tuya:
-    [22:03:11][C][tuya:032]:   Datapoint 1: switch (value: ON)
-    [22:03:11][C][tuya:032]:   Datapoint 2: switch (value: OFF)
-    [22:03:11][C][tuya:034]:   Datapoint 3: int value (value: 20)
-    [22:03:11][C][tuya:034]:   Datapoint 4: int value (value: 19)
-    [22:03:11][C][tuya:034]:   Datapoint 5: int value (value: 0)
-    [22:03:11][C][tuya:036]:   Datapoint 7: enum (value: 1)
-    [22:03:11][C][tuya:046]:   Product: '{"p":"ynjanlglr4qa6dxf","v":"1.0.0","m":0}'
+    [10:07:48][C][tuya:041]: Tuya:
+    [10:07:48][C][tuya:056]:   Datapoint 1: switch (value: OFF)
+    [10:07:48][C][tuya:058]:   Datapoint 2: int value (value: 25)
+    [10:07:48][C][tuya:062]:   Datapoint 4: enum (value: 0)
+    [10:07:48][C][tuya:062]:   Datapoint 5: enum (value: 4)
+    [10:07:48][C][tuya:056]:   Datapoint 8: switch (value: OFF)
+    [10:07:48][C][tuya:056]:   Datapoint 101: switch (value: OFF)
+    [10:07:48][C][tuya:058]:   Datapoint 3: int value (value: 31)
+    [10:07:48][C][tuya:056]:   Datapoint 104: switch (value: ON)
+    [10:07:48][C][tuya:056]:   Datapoint 106: switch (value: OFF)
+    [10:07:48][C][tuya:056]:   Datapoint 107: switch (value: OFF)
+    [10:07:48][C][tuya:056]:   Datapoint 103: switch (value: OFF)
+    [10:07:48][C][tuya:076]:   Product: 'stgbzbctueolecwx1.0.0'
 
 On this controller, the data points are:
 
 - 1 represents the climate on/off state.
-- 2 represents the child lock switch. (use the :doc:`/components/switch/tuya` component to control this)
-- 3 represents the target temperature.
-- 4 represents the current temperature.
-- 5 represents the timer but is not yet available to be used in ESPHome.
-- 7 represents the eco mode switch.
+- 2 represents the target temperature.
+- 3 represents the current temperature
+- 4 represents the active state (operating mode, such as heat/cool). This is a value that can range from 0-4 on this device. Each value is a different mode.
+- 5 represents the Fan Speed / Fan Mode. This is a value that can range from 0-4 on this device. Each value is a different speed.
+- 8 represents the Eco Mode preset.
+- 101 represents the Sleep Mode preset.
+- 103 represents the Auto-Clean feature. (use the :doc:`/components/switch/tuya` component to control this)
+- 104 represents the display toggle (on device) feature. (use the :doc:`/components/switch/tuya` component to control this)
+- 106 represents the Vertical Swing switch.
+- 107 represents the Horizontal Swing switch.
```

Comment:
We dont really need to change this block, its just one example.

If you want to add an example for a specific device, it should go on https://devices.esphome.io

Reply:
Agreed, I was just "updating" what was there, and was considering it to be a bit out of date.

Reply:
I didnt mean delete the whole block... 
I just meant dont change it so much...haha

Reply:
Ah ok, I will fix this. sorry for the delay. I have been pretty tied up lately.